service/src/main/java/gov/nasa/pds/api/registry/business/BundleDAO.java

```diff
@@ -25,26 +33,22 @@
  */
 public class BundleDAO
 {
-    private OpenSearchRegistryConnection esConnection;
-    
-    /**
-     * Constructor
-     * @param esConnection opensearch connection
-     */
-    public BundleDAO(OpenSearchRegistryConnection esConnection)
+    @SuppressWarnings("unused")
+	private static final Logger log = LoggerFactory.getLogger(BundleDAO.class);
+
+    static public Map<String,String> searchConstraints()
     {
-        this.esConnection = esConnection;
+    	Map<String,String> preset = new HashMap<String,String>();
+    	preset.put("product_class", "Product_Bundle");
+    	return preset;
     }
-    
-    
-    public void getBundleCollections(String lidvid, int start, int limit, List<String> fields, 
-            List<String> sort, boolean onlySummary) throws IOException, LidVidNotFoundException
+
+    static public void getBundleCollections() throws IOException, LidVidNotFoundException
     {
         // TODO: Move code from the bundle controller here.
```

Comment:
Is it a regression in the capabilities of the API ?

Reply:
I am asking since we want to deliver a version 1 this week

Reply:
No. In fact, it works as documented now where as it did not before.